CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.8.1 (2020-02-20)
+
+- Fixed the `help` property in the schema to only accept values of type `object`. [#41](https://github.com/blackbaud/skyux-config/pull/41)
+- Fixed typo in the schema for the `bbCheckout` property. [#40](https://github.com/blackbaud/skyux-config/pull/40)
```

Comment:
Add an "a" before typo and you're golden.

Reply:
Added